docs(v1.8.0): update important note (#1041)
longhorn/website: content/docs/1.8.0/important-notes/_index.md

- Re-add the disk to `node.spec.disks` with the updated configuration.
- Restore the v2 volumes.

### Potential Volume and Backup Data Corruption Prior to v1.8.0

The checksum of files inside a v2 volume with multiple replicas may continuously change prior to version 1.8.0. This issue occurs because SPDK allocates clusters without initialization, leading to data inconsistencies across replicas. The varying data read from the volume can result in data corruption and broken backups. This issue has been resolved in version 1.8.0. For more information, see Longhorn Issue [#10135](https://github.com/longhorn/longhorn/issues/10135).

### Support Configurable CPU Cores

> Longhorn supports configurable CPU cores for the v2 data engine, offering both global and per-node configuration options from v1.8.0, providing greater control and flexibility for optimizing performance and resource allocation.

Reviewer: ```suggestion
Longhorn v1.8.0 supports [configurable CPU cores](../v2-data-engine/features/configurable-cpu-cores) for the V2 Data Engine. The global and node-specific configuration options provide greater control and flexibility for optimizing performance and resource allocation.
```

> For more information, see [Configurable CPU Cores](../v2-data-engine/features/configurable-cpu-cores).

Reviewer: Please remove this. I added the link to the previous paragraph.
